tests/ReplayData/PullRequestReview.setUp.txt

```diff
@@ -18,7 +18,7 @@ None
 None
```

Comment:
why are these two replay data files changing? We ave no changes in test files and github files only touch typing.

Reply:
There are some code in these 2 replay data and change when running global file replace.

Reverted.